aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/UpdateHandlerTest.java

```diff
@@ -204,45 +413,45 @@ public void itCallsGetPolicyAndPutPermission() {
     class WhenThereIsAnException {
         @BeforeEach
         public void setup() {
-            request =  makeValidRequest();
+            request = makeValidRequest();
         }
 
         @Test
         public void itThrowsConflictException() {
             doThrow(ConflictException.builder().build())
-                .when(proxy).injectCredentialsAndInvokeV2(any(), any());
+                    .when(proxy).injectCredentialsAndInvokeV2(any(), any());
 
             assertThrows(CfnAlreadyExistsException.class, () -> subject.handleRequest(proxy, request, null, logger));
         }
 
         @Test
         public void itThrowsNotFoundException() {
             doThrow(ResourceNotFoundException.builder().build())
-                .when(proxy).injectCredentialsAndInvokeV2(any(), any());
+                    .when(proxy).injectCredentialsAndInvokeV2(any(), any());
 
             assertThrows(CfnNotFoundException.class, () -> subject.handleRequest(proxy, request, null, logger));
         }
 
         @Test
         public void itThrowsInternalServerException() {
             doThrow(InternalServerException.builder().build())
-                .when(proxy).injectCredentialsAndInvokeV2(any(), any());
+                    .when(proxy).injectCredentialsAndInvokeV2(any(), any());
 
             assertThrows(CfnServiceInternalErrorException.class, () -> subject.handleRequest(proxy, request, null, logger));
         }
 
         @Test
         public void itThrowsThrottlingException() {
             doThrow(ThrottlingException.builder().build())
-                .when(proxy).injectCredentialsAndInvokeV2(any(), any());
+                    .when(proxy).injectCredentialsAndInvokeV2(any(), any());
 
             assertThrows(CfnThrottlingException.class, () -> subject.handleRequest(proxy, request, null, logger));
         }
 
         @Test
         public void itThrowsValidationException() {
             doThrow(ValidationException.builder().build())
-                .when(proxy).injectCredentialsAndInvokeV2(any(), any());
+                    .when(proxy).injectCredentialsAndInvokeV2(any(), any());
 
             assertThrows(CfnInvalidRequestException.class, () -> subject.handleRequest(proxy, request, null, logger));
         }
```

Comment:
[There is still quite a lot of non-relevant diff noise :(](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeguru-profiler/pull/16/#discussion_r451589067). Please consider removing it.